website/content/docs/release-notes/v0_12_0.mdx

```diff
@@ -0,0 +1,81 @@
+---
+layout: docs
+page_title: v0.12.0
+description: |-
+  Boundary release notes for v0.12.0
+---
+
+# Boundary v0.12.0
+
+The release notes below contain information about new functionality available in the Boundary v0.12.0 release.
+To see a granular record of when each item was merged into the Boundary project, please refer to the [Changelog](https://github.com/hashicorp/boundary/blob/main/CHANGELOG.md).
+To learn about what Boundary consists of, we highly recommend you review the [Getting Started Page](/boundary/docs/getting-started).
+
+For instructions on how to upgrade an existing Boundary deployment to v0.12.0, refer to Boundary's [general upgrade guide](/boundary/tutorials/oss-configuration/upgrade-version).
+
+## Boundary v0.12.0 highlights
+
+The 0.12.0 release adds major new functionality to Boundary.
+Highlights include:
+
+- Addresses on targets
+- Authentication improvements
+
+## New features
+
+**Addresses on targets**: HashiCorp Boundary offers an extensible domain model that allows administrators to organize target resources in a way that best compliments how their organization manages its computing infrastructure.
+However, that flexibility could sometimes become a hindrance when it comes to setting up a quick proof-of-concept, and defining a target to create a session.
+
+With version 0.12.0, we are introducing a quick way to set up a target.
+Previously, you would need to define [hosts](/boundary/docs/concepts/domain-model/hosts) within a [host set](/boundary/docs/concepts/domain-model/host-sets), and then assign that to a [target](/boundary/docs/concepts/domain-model/targets).
```

Comment:
```suggestion
Previously, you would need to define [hosts](/boundary/docs/concepts/domain-model/hosts) within a [host set](/boundary/docs/concepts/domain-model/host-sets) and then assign that to a [target](/boundary/docs/concepts/domain-model/targets).
```

Reply:
I think this sentence might need the comma, because it is 2 independent clauses joined by a coordinating conjunction.

Reply:
Definitely, thanks for correcting me on this 👍